resteasy-rxjava/src/test/java/org/jboss/resteasy/rxjava/RxTest.java

```diff
@@ -67,13 +67,13 @@ public void testSingle()
       String data = client.target(generateURL("/single")).request().get(String.class);
       assertEquals("got it", data);
 
-      String[] data2 = client.target(generateURL("/observable")).request().get(String[].class);
-      assertArrayEquals(new String[] {"one", "two"}, data2);
+//      String[] data2 = client.target(generateURL("/observable")).request().get(String[].class);

```

Comment:
Is it normal that these two tests were commented?

Reply:
I've updated an re-enabled them in PR #1493.